.github/workflows/upgrade_test.yaml

```diff
@@ -0,0 +1,71 @@
+name: Upgrade functionality test
+
+on: 
+  push:
+    branches: [ upgrade-testing ]
+
+jobs:
+  
+  upgrade-functionality-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout action sources
+      uses: actions/checkout@v2.3.4
+
+    - name: Checkout fission sources
+      uses: actions/checkout@v2.3.4
+      with:
+        repository: fission/examples
+        path: examples
+
+    - name: Setup go
+      uses: actions/setup-go@v2.1.3
+      with:
+        go-version: '1.15.12'
+
+    - name: Setup Helm
+      uses: Azure/setup-helm@v1
+      with:
+        version: v3.3.4
+    
+    - name: Update helm
+      run: helm repo add helm https://charts.helm.sh/stable 
```

Comment:
This could specific to releases before 1.13. With the latest helm, the stable repo is now deprecated. If we still need this, we should move it `install_stable_release` function.

Reply:
Removed helm upgrade and tested the workflow, nothing breaks.